common/changes/@typespec/versioning/fix-versioning-across-namespaces_2023-03-31-18-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/versioning",
+      "comment": "Fix: Issue with using version as a template parameter across different namespace. Include a significant change in the versioning library internals.",
+      "type": "none"
```

Comment:
I think we need to be more explicit here as there seems to be various breaking changes to exported method names.

Reply:
I guess the only one that really changing is `getAvailabilityMap`